bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/SuuToChiChi_FullyHadronic/SuuToChiChi_FullyHadronic_MSuu8TeV_MChi3TeV/SuuToChiChi_FullyHadronic_MSuu8TeV_MChi3TeV_proc_card.dat

```diff
@@ -0,0 +1,73 @@
+#************************************************************
+#*                     MadGraph5_aMC@NLO                    *
+#*                                                          *
+#*                *                       *                 *
+#*                  *        * *        *                   *
+#*                    * * * * 5 * * * *                     *
+#*                  *        * *        *                   *
+#*                *                       *                 *
+#*                                                          *
+#*                                                          *
+#*         VERSION 2.7.3                 2020-06-21         *
+#*                                                          *
+#*    The MadGraph5_aMC@NLO Development Team - Find us at   *
+#*    https://server06.fynu.ucl.ac.be/projects/madgraph     *
+#*                                                          *
+#************************************************************
+#*                                                          *
+#*               Command File for MadGraph5_aMC@NLO         *
+#*                                                          *
+#*     run as ./bin/mg5_aMC  filename                       *
+#*                                                          *
+#************************************************************
+set default_unset_couplings 99
+set group_subprocesses Auto
+set ignore_six_quark_processes False
+set loop_optimized_output True
+set loop_color_flows False
+set gauge unitary
+set complex_mass_scheme False
+set max_npoint_for_channel 0
+import model sm
+define p = g u c d s u~ c~ d~ s~
+define j = g u c d s u~ c~ d~ s~
+define l+ = e+ mu+
+define l- = e- mu-
+define vl = ve vm vt
+define vl~ = ve~ vm~ vt~
+import model diquarkVLQ2020_UFO
+generate p p > suu >  W+ b H t, W+ > j j, (t > W+ b, W+ > j j),(H > b \
+b~)
+add process p p > suu >  W+ b H t, W+ > j j, (t > W+ b, W+ > j j),(H >\
+ Z Z, Z > j j , Z > j j)
```

Comment:
I had not realized that j was defined for only u,d,c,s,u~,d~,s~,c~ . We do want to have the decays to b quarks included, so I defined a new multiparticle j0 = u d c s b u~ d~ c~ s~ b~ and had the W and Z bosons decay to j0 j0. Obviously this increases the complexity of the diagrams, and it now takes over an hour for Madgraph to compile and run. I also see that the output MG folder is 20+ gb for generated 5000 events, most of which is subprocesses folder. Is this the correct way to apply this change?

Reply:
what's the size of output before the output, the change should not make too large difference since only only one decay Z > b b~ is included.

Reply:
I have tested doing the event generation with and without including b and b~ in the definition for j0. When I define j0 = g u d c s b u~ d~ c~ s~ b~ and launch, it results in 22 GB of sub-processes, whereas this is only 12 GB when j0 =  g u d c s u~ d~ c~ s~. This definition is the only difference between MG commands in the two tests.

Reply:
Hi, I can't do it local since "launch" the process directly kill my laptop... could you please check the directories under "SubProcesses", what's the extra directories of Z>j5j5

Reply:
Hi, these are all the folders that I get when I use the j0 definition w/ b and b~

**with Z>qq component**

P10_qq_htht_h_zz_z_qq_z_qq_t_wpb_wp_qq_h_zz_z_bbx_z_bbx_t_wpb_wp_qq
P10_qq_htht_h_zz_z_qq_z_bbx_t_wpb_wp_qq_h_zz_z_qq_z_qq_t_wpb_wp_qq
P10_qq_htht_h_zz_z_qq_z_qq_t_wpb_wp_qq_h_zz_z_qq_z_bbx_t_wpb_wp_qq
P10_qq_htht_h_zz_z_qq_z_bbx_t_wpb_wp_qq_h_zz_z_bbx_z_bbx_t_wpb_wp_qq
P10_qq_htht_h_zz_z_qq_z_bbx_t_wpb_wp_qq_h_zz_z_qq_z_bbx_t_wpb_wp_qq
P10_qq_htht_h_zz_z_qq_z_bbx_t_wpb_wp_qq_h_wpwm_wp_qq_wm_qq_t_wpb_wp_qq
P6_qq_htzt_h_wpwm_wp_qq_wm_qq_t_wpb_wp_qq_z_qq_t_wpb_wp_qq
P10_qq_htht_h_zz_z_qq_z_qq_t_wpb_wp_qq_h_wpwm_wp_qq_wm_qq_t_wpb_wp_qq
P10_qq_htht_h_zz_z_qq_z_qq_t_wpb_wp_qq_h_zz_z_qq_z_qq_t_wpb_wp_qq
P8_qq_htht_h_zz_z_qq_z_qq_t_wpb_wp_qq_h_zz_z_bbx_z_bbx_t_wpb_wp_qq
P8_qq_htht_h_zz_z_qq_z_bbx_t_wpb_wp_qq_h_zz_z_qq_z_qq_t_wpb_wp_qq
P8_qq_htht_h_zz_z_qq_z_bbx_t_wpb_wp_qq_h_zz_z_qq_z_bbx_t_wpb_wp_qq
P8_qq_htht_h_zz_z_qq_z_bbx_t_wpb_wp_qq_h_zz_z_bbx_z_bbx_t_wpb_wp_qq
P8_qq_htht_h_zz_z_qq_z_qq_t_wpb_wp_qq_h_zz_z_qq_z_bbx_t_wpb_wp_qq
P8_qq_htht_h_zz_z_qq_z_qq_t_wpb_wp_qq_h_zz_z_qq_z_qq_t_wpb_wp_qq
P11_qq_htht_h_zz_z_qq_z_qq_t_wpb_wp_qq_h_zz_z_bbx_z_bbx_t_wpb_wp_qq
P11_qq_htht_h_zz_z_qq_z_bbx_t_wpb_wp_qq_h_zz_z_qq_z_qq_t_wpb_wp_qq
P11_qq_htht_h_zz_z_qq_z_bbx_t_wpb_wp_qq_h_zz_z_qq_z_bbx_t_wpb_wp_qq
P11_qq_htht_h_zz_z_qq_z_bbx_t_wpb_wp_qq_h_zz_z_bbx_z_bbx_t_wpb_wp_qq
P15_qq_ztzt_z_qq_t_wpb_wp_qq_z_bbx_t_wpb_wp_qq
P11_qq_htht_h_zz_z_qq_z_qq_t_wpb_wp_qq_h_zz_z_qq_z_bbx_t_wpb_wp_qq
P11_qq_htht_h_zz_z_qq_z_qq_t_wpb_wp_qq_h_zz_z_qq_z_qq_t_wpb_wp_qq
P8_qq_htht_h_bbx_t_wpb_wp_qq_h_zz_z_qq_z_qq_t_wpb_wp_qq
P8_qq_htht_h_bbx_t_wpb_wp_qq_h_zz_z_qq_z_bbx_t_wpb_wp_qq
P5_qq_htzt_h_zz_z_qq_z_qq_t_wpb_wp_qq_z_qq_t_wpb_wp_qq
P5_qq_htzt_h_zz_z_qq_z_qq_t_wpb_wp_qq_z_bbx_t_wpb_wp_qq
P5_qq_htzt_h_zz_z_qq_z_bbx_t_wpb_wp_qq_z_qq_t_wpb_wp_qq
P5_qq_htzt_h_zz_z_qq_z_bbx_t_wpb_wp_qq_z_bbx_t_wpb_wp_qq
P5_qq_htzt_h_zz_z_bbx_z_bbx_t_wpb_wp_qq_z_qq_t_wpb_wp_qq
P4_qq_htzt_h_bbx_t_wpb_wp_qq_z_qq_t_wpb_wp_qq
P2_qq_wpbht_wp_qq_h_zz_z_qq_z_qq_t_wpb_wp_qq
P2_qq_wpbht_wp_qq_h_zz_z_qq_z_bbx_t_wpb_wp_qq
P15_qq_ztzt_z_qq_t_wpb_wp_qq_z_qq_t_wpb_wp_qq
P14_qq_wpbzt_wp_qq_z_qq_t_wpb_wp_qq

**no Z>qq component**
P6_qq_htzt_h_wpwm_wp_qq_wm_qq_t_wpb_wp_qq_z_bbx_t_wpb_wp_qq
P10_qq_htht_h_zz_z_bbx_z_bbx_t_wpb_wp_qq_h_wpwm_wp_qq_wm_qq_t_wpb_wp_qq
P10_qq_htht_h_wpwm_wp_qq_wm_qq_t_wpb_wp_qq_h_wpwm_wp_qq_wm_qq_t_wpb_wp_qq
P3_qq_wpbht_wp_qq_h_wpwm_wp_qq_wm_qq_t_wpb_wp_qq
P8_qq_htht_h_zz_z_bbx_z_bbx_t_wpb_wp_qq_h_zz_z_bbx_z_bbx_t_wpb_wp_qq
P9_qq_htht_h_bbx_t_wpb_wp_qq_h_wpwm_wp_qq_wm_qq_t_wpb_wp_qq
P9_qq_htht_h_wpwm_wp_qq_wm_qq_t_wpb_wp_qq_h_wpwm_wp_qq_wm_qq_t_wpb_wp_qq
P14_qq_wpbzt_wp_qq_z_bbx_t_wpb_wp_qq
P15_qq_ztzt_z_bbx_t_wpb_wp_qq_z_bbx_t_wpb_wp_qq
P9_qq_htht_h_bbx_t_wpb_wp_qq_h_bbx_t_wpb_wp_qq
P12_qq_htht_h_wpwm_wp_qq_wm_qq_t_wpb_wp_qq_h_wpwm_wp_qq_wm_qq_t_wpb_wp_qq
P11_qq_htht_h_zz_z_bbx_z_bbx_t_wpb_wp_qq_h_zz_z_bbx_z_bbx_t_wpb_wp_qq
P8_qq_htht_h_bbx_t_wpb_wp_qq_h_zz_z_bbx_z_bbx_t_wpb_wp_qq
P8_qq_htht_h_bbx_t_wpb_wp_qq_h_bbx_t_wpb_wp_qq
P7_qq_htht_h_bbx_t_wpb_wp_qq_h_bbx_t_wpb_wp_qq
P5_qq_htzt_h_zz_z_bbx_z_bbx_t_wpb_wp_qq_z_bbx_t_wpb_wp_qq
P2_qq_wpbht_wp_qq_h_zz_z_bbx_z_bbx_t_wpb_wp_qq
P1_qq_wpbht_wp_qq_h_bbx_t_wpb_wp_qq
P13_qq_wpbwpb_wp_qq_wp_qq
P10_qq_htht_h_zz_z_bbx_z_bbx_t_wpb_wp_qq_h_zz_z_bbx_z_bbx_t_wpb_wp_qq
P4_qq_htzt_h_bbx_t_wpb_wp_qq_z_bbx_t_wpb_wp_qq